declarative/persistence/pom.xml

```diff
@@ -29,6 +29,7 @@
             <groupId>${project.groupId}</groupId>
             <artifactId>blaze-expression-persistence</artifactId>
             <version>${project.version}</version>
+            <scope>provided</scope>
```

Comment:
This one shouldn't be necessary

Reply:
As in, the entire dependency, or the scope?

Reply:
The scope

Reply:
Fixed